tests/test_E1_process_results.py

```diff
@@ -1,6 +1,8 @@
 # from _constants import JSON_PATH
```

Comment:
Use of messages with `assert` calls ?

Reply:
To be done in another pr, connected to #540